src/frontend/screens/ExportObservations.tsx

```diff
@@ -0,0 +1,210 @@
+import {HeaderText} from '../sharedComponents/Text/HeaderText';
+import {defineMessages, useIntl} from 'react-intl';
+import {BodyText} from '../sharedComponents/Text/BodyText';
+import {PrimaryButton, SecondaryButton} from '../sharedComponents/Buttons';
+import {NativeRootNavigationProps} from '../sharedTypes/navigation';
+import {DARK_GREY, LIGHT_GREY, WARNING_RED} from '../lib/styles';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import {useState} from 'react';
+import {useActiveProject} from '../contexts/ActiveProjectContext';
+import {UIActivityIndicator} from 'react-native-indicators';
+import {useObservations} from '../hooks/server/observations';
+import {useTracks} from '../hooks/server/track';
+import * as Sentry from '@sentry/react-native';
+import {useExportObservationsAndShare} from '../hooks/server/projects';
+
+const m = defineMessages({
+  close: {
+    id: 'screens.ExportObservations.close',
+    defaultMessage: 'Close',
+  },
+  download: {
+    id: 'screens.ExportObservations.download',
+    defaultMessage: 'Download',
+  },
+  allObservations: {
+    id: 'screens.ExportObservations.allObservations',
+    defaultMessage: 'All Observations',
+  },
+  allObservationsDescription: {
+    id: 'screens.ExportObservations.allObservationsDescription',
+    defaultMessage: 'Text only as a GeoJson file',
+  },
+  allObservationsAndMedia: {
+    id: 'screens.ExportObservations.allObservationsAndMedia',
+    defaultMessage: 'All Observations with Media',
+  },
+  allObservationsAndMediaDescription: {
+    id: 'screens.ExportObservations.allObservationsAndMediaDescription',
+    defaultMessage: 'Images and audio in a .zip file',
+  },
+  tracks: {
+    id: 'screens.ExportObservations.tracks',
+    defaultMessage: 'Tracks',
+  },
+  tracksDescription: {
+    id: 'screens.ExportObservations.tracksDescription',
+    defaultMessage: 'Tracks only as a GeoJson file',
+  },
+  errorMessage: {
+    id: 'screens.ExportObservations.errorMessage',
+    defaultMessage: 'Choose an option',
+  },
+});
+
+export type Exports = 'Observation' | 'Tracks' | 'ObservationsWithMedia';
+
+export const ExportObservations = ({
+  navigation,
+}: NativeRootNavigationProps<'ExportObservations'>) => {
+  const {formatMessage} = useIntl();
+  const [typeToExport, setTypeToExport] = useState<null | Exports>(null);
+  const [showError, setShowError] = useState<boolean>(false);
+  const {data: observations} = useObservations();
+  const {data: tracks} = useTracks();
+  const {projectId} = useActiveProject();
+  const exportAndShare = useExportObservationsAndShare({projectId});
+
+  async function handlePressDownload() {
+    if (!typeToExport) {
+      setShowError(true);
+      return;
+    }
+    exportAndShare.mutate(
+      {exportType: typeToExport},
+      {
+        onError: err => {
+          Sentry.captureException(err);
+          navigation.navigate('ErrorBottomSheet');
+        },
+      },
```

Comment:
i might've missed this is the user requirements, but should there but a success handler here to navigate away from this screen?

Reply:
ah nevermind, the notes in the issue under the `Export selector` clarify this.